contrib/examples/cluster-operator-roles-template.yaml

```diff
@@ -8,11 +8,10 @@
 # itself.
 #
 ########
-apiVersion: v1
+apiVersion: template.openshift.io/v1
 kind: Template
 metadata:
-  name: cluster-operator-deploy-template
-  namespace: ${CLUSTER_OPERATOR_NAMESPACE}
```

Comment:
Is removing this namespace needed because the template itself is being created, rather than just processed from a file?

Reply:
At one point the plan was to create the template but that did not pan out, it went back to being processed only as a file. We could probably restore namespace here if there's any reason to.